.github/workflows/tests.yaml

```diff
@@ -18,24 +18,14 @@ jobs:
           channel: 1.26-strict/stable
           juju-channel: 3.1
           bootstrap-options: '--agent-version=3.1.0'
-          # provide a pool of at least 2 IP addresses to the metallb addon
+          # provide a pool of at least 3 IP addresses to the metallb addon
           microk8s-addons: "dns hostpath-storage metallb:10.64.140.43-10.64.140.49"
 
       - name: Run integration tests
         run: tox -e integration -- --model testing
-        env:
-          KRATOS_EXTERNAL_IDP_CLIENT_ID: ${{ secrets.KRATOS_EXTERNAL_IDP_CLIENT_ID }}
-          KRATOS_EXTERNAL_IDP_TENANT_ID: ${{ secrets.KRATOS_EXTERNAL_IDP_TENANT_ID }}
-          KRATOS_EXTERNAL_IDP_CLIENT_SECRET: ${{ secrets.KRATOS_EXTERNAL_IDP_CLIENT_SECRET }}
 
-      # Print out logs and metadata to assist in troubleshooting
-      - name: Get traefik-admin juju logs
-        run: juju debug-log --replay --include traefik-admin/0
-        if: failure()
-
-      - name: Get traefik-public juju logs
-        run: juju debug-log --replay --include traefik-public/0
-        if: failure()
+      - name: Print hydra logs for debugging
+        run: kubectl logs -n testing -c hydra hydra-0
```

Comment:
Would you suggest adding all of our charm's logs? I was even thinking of removing this, because I added it for debugging purposes

Reply:
I would add it for debugging purposes, in case of failure. We have the scheduled tests - if they fail it would be easier to spot the bug having the logs.